main.nf

```diff
@@ -69,66 +77,83 @@ nxtflow_libs=file(params.rlocation)
 log.info "===================================="
 log.info " RNAbp : Small RNA-Seq Best Practice v${version}"
 log.info "===================================="
-log.info "Reads        : ${params.input}"
-log.info "Genome       : ${params.genome}"
-log.info "Index        : ${params.index}"
-log.info "Annotation   : ${params.gtf}"
-log.info "miRBase mature  : ${params.bowtie_miRBase_mature}"
-log.info "miRBase hairpin : ${params.bowtie_miRBase_hairpin}"
-log.info "Rfam         : ${params.bowtie_rfam}"
-log.info "Current home : $HOME"
-log.info "Current user : $USER"
-log.info "Current path : $PWD"
-log.info "R libraries  : ${params.rlocation}"
-log.info "Script dir   : $baseDir"
-log.info "Working dir  : $workDir"
-log.info "Output dir   : ${params.out}"
+log.info "Reads                : ${params.reads}"
+log.info "Genome               : ${params.genome}"
+log.info "Bowtie2 Index        : ${params.index}"
+log.info "Annotation           : ${params.gtf}"
+log.info "miRBase mature       : ${params.mature}"
+log.info "miRBase hairpin      : ${params.hairpin}"
+log.info "Current home         : $HOME"
+log.info "Current user         : $USER"
+log.info "Current path         : $PWD"
+log.info "R libraries          : ${params.rlocation}"
+log.info "Script dir           : $baseDir"
+log.info "Working dir          : $workDir"
+log.info "Output dir           : ${params.outdir}"
 log.info "===================================="
 
 // Create R library directories if not already existing
 nxtflow_libs.mkdirs()
 
 // Set up nextflow objects
-params.gtf = file(params.gtf)
-params.bed12 = file(params.bed12)
-params.bowtie2 = file(params.bowtie2)
-params.bowtie_miRBase_mature = file(params.bowtie_miRBase_mature)
-params.bowtie_miRBase_hairpin = file(params.bowtie_miRBase_hairpin)
-
-// Validate inputs
-if( !bowtie_miRBase_mature.exists() ) exit 1, "Missing Bowtie 1 miRBase_mature index: ${bowtie_miRBase_mature}"
-if( !bowtie_miRBase_hairpin.exists() ) exit 1, "Missing Bowtie 1 miRBase_hairpin index: ${bowtie_miRBase_hairpin}"
-if( !bowtie2.exists() ) exit 1, "Missing Bowtie 2 index: ${bowtie2}"
-if( !gtf.exists() )   exit 2, "Missing GTF annotation: ${gtf}"
-if( !bed12.exists() ) exit 2, "Missing BED12 annotation: ${bed12}"
+gtf = file(params.gtf)
+bed12 = file(params.bed12)
+index = file(params.index)
+mature = file(params.mature)
+hairpin = file(params.hairpin)
+
+// Validate inputs(NOT WORKING!!)
+// if( !mature.exists() ) exit 1, "Missing Bowtie 1 miRBase_mature index: ${mature}"
+// if( !hairpin.exists() ) exit 1, "Missing Bowtie 1 miRBase_hairpin index: ${hairpin}"
+// if( !index.exists() ) exit 1, "Missing Bowtie 2 index: ${index}"
+// if( !gtf.exists() )   exit 2, "Missing GTF annotation: ${gtf}"
+// if( !bed12.exists() ) exit 2, "Missing BED12 annotation: ${bed12}"
```

Comment:
These existence check didn't work. I looked into the Nextflow document and haven't found a function for this purpose. Thus I deleted them so that the scripts can be run.